packages/webapp/src/containers/Finances/NewExpense/AddExpense/styles.module.scss

```diff
@@ -57,23 +57,13 @@
     letter-spacing: 0.324px;
     color: #4d4d4d;
   }
-  input {
+  input, textarea {
     width: 60%;
-    height: 37px;
-    border: 1px solid #efefef;
-    box-sizing: border-box;
-    border-radius: 4px;
-    padding: 0 5%;
-  }
-  .textArea {
-    font-size: 16px;
-    line-height: 24px;
-    font-family: 'Open Sans', 'SansSerif', serif;
```

Comment:
I think we should keep font-size and font-family. Otherwise...
![Screenshot 2023-05-02 at 1 44 46 PM](https://user-images.githubusercontent.com/33141219/235781799-055e1b23-73b6-4130-9e27-cafb47d812b1.png)

```
  textarea {
    font-size: 16px;
    font-family: 'Open Sans', 'SansSerif', serif;
  }
```

Reply:
Isn't this normally set at the top level component instead of individual inputs? and since this needs a refactor to use react-hook-forms will that fix it also?

Reply:
@Duncan-Brain Yeah font-family is usually inherited, but for a textarea, most browsers apply monospace font 😁 So you will have to add it in manually to reverse that. I don't think the font size is 16px though... if it's going to be kept equivalent to Edit Expense I think the default font-size is ok. My browser is calculating that at 13.33px (??!) but maybe check with yours.

Reply:
I agree, they should not be written over and over again. But `InputAutoSize` also needed them, so I thought global styling was not working.
I'm not sure how the refactor will change the input, but I think we could change the font-family for the patch? The font is `Arial` in beta which is probably wrong, but `monospace` looks very different.

Reply:
Font-size: both are inherited

Font-family: done